### PR TITLE
added new set of test cases for secgroups deny rules changes and reve…

### PR DESCRIPTION
**Pull request by Bhavanaashok33**
…rted the older test case

### CODE REVIEW

File: agent-ovs/lib/PolicyManager.cpp [hunk 1]
```diff
@@ -1157,11 +1158,13 @@ static bool updatePolicyRules(OFFramework& framework,
                     RedirectDestGroup::resolve(framework, destGrpUri.get());
                     newRedirGrps.insert(destGrpUri.get());
                 }
-                optional<shared_ptr<modelgbp::gbp::LogAction> > resloveLog = 
+	        else if(r->getTargetClass().get() == LogAction::CLASS_ID) {
+		    optional<shared_ptr<modelgbp::gbp::LogAction> > resloveLog =
```
Comment (tomflynn) [inline agent-ovs/lib/PolicyManager.cpp:1162]: resloveLog -> resolveLog and the indentation is messed up here

File: agent-ovs/ovs/test/AccessFlowManager_test.cpp [hunk 2]
```diff
@@ -359,6 +350,77 @@ BOOST_FIXTURE_TEST_CASE(secGrp, AccessFlowManagerFixture) {
...
+           ->setAddress("192.169.0.0")	
+           .setPrefixLen(16);	
+
+	shared_ptr<modelgbp::gbp::SecGroupRule> r1, r2, r3, r4;	
+        //secgrp 3	
+	secGrp3 = space->addGbpSecGroup("secgrp3");		
+        //action 1	
+	action1 = space->addGbpAllowDenyAction("action1");	
+	action1->setAllow(0).setOrder(5);	
+        action2 =  space->addGbpLogAction("action2");
```
Comment (tomflynn) [inline agent-ovs/ovs/test/AccessFlowManager_test.cpp:370]: nit: can you use consistent indentation here

**Review by tomflynn — COMMENTED** [review]
Some minor formatting issue but it looks ok to me.

File: agent-ovs/lib/PolicyManager.cpp (hunk 1 above)
Comment (Bhavanaashok33) [reply agent-ovs/lib/PolicyManager.cpp:1162]: okay fix it

File: agent-ovs/ovs/test/AccessFlowManager_test.cpp (hunk 2 above)
Comment (Bhavanaashok33) [reply agent-ovs/ovs/test/AccessFlowManager_test.cpp:370]: okay will fix as well

Comment (Bhavanaashok33) [reply agent-ovs/ovs/test/AccessFlowManager_test.cpp]: couple of tabs were present. Have removed them.

**Review by tomflynn — APPROVED** [review]

**Pull request merged**